test/models/subscription_test.rb

```diff
@@ -7,6 +7,32 @@ def setup
       @other = katello_subscriptions(:other_subscription)
     end
 
+    def test_import_all
```

Comment:
This path is not broken on master, but Subscription also uses import_all so I decided to be safe and add test coverage now